### ISSUE TITLE
Exceptionbehandlung: Doppelte Posts

### ISSUE THREAD

**Issue by hansmorb** [wielebenwir/commonsbooking#1189]
https://github.com/wielebenwir/commonsbooking/blob/7f3ef952eb0eb68c88a2578a1ed346d482c7f279/src/Migration/Migration.php#L345

Bei der Migration wird anscheinend geprüft, ob der Post der erstellt werden sollte schonmal erstellt wurde. Wenn die Migration aber irgendwo abbricht und ungünstig neu gestartet wird, dann können u.U. zwei Posts mit dem gleichen Namen entstehen. Das führt zu einer unbehandelten Ausnahme bei der Migration. 

Vorschlag: Bei so einem Fall einfach beide Posts löschen und neu anlegen. Was denkt ihr?

**Comment by chriwen** [comment]
Ja. Find ich gut. Ich hoffe es gibt nicht mehr all zu viele Migrationen 😉
